plop-templates/composite-component.hbs

```diff
@@ -0,0 +1,51 @@
+import classnames from 'classnames';
+
+import { downcastRef } from '../../util/typing';
+
+/**
+ * @typedef {import('../../types').CompositeProps} CompositeProps
+ * @typedef {Omit<import('preact').JSX.HTMLAttributes<HTMLElement>, 'size'>} HTMLAttributes
+ *
+ * @typedef {{ name }}Props
+ * @prop {'sm'|'md'|'lg'} [size='md']
+ * @prop {'primary'|'secondary'} [variant='secondary']
+ */
+
+/**
+ * TODO
+ *
+ * @param {CompositeProps & {{name}}Props & Omit<HTMLAttributes, 'size'>} props
+ */
+const {{name}}Next = function {{name}}({
```

Comment:
I presume these templates are going to be revised to remove the `const {ComponentName}Next` wrapper once the legacy components have been retired and there are no name conflicts?

Reply:
Yep, will be added to the list of things to update https://github.com/hypothesis/frontend-shared/issues/566